infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/impl/AbstractSszMutableComposite.java

```diff
@@ -239,6 +239,15 @@ public SszMutableComposite<SszChildT> createWritableCopy() {
    */
   protected abstract void checkIndex(int index, boolean set);
 
+  /**
+   * Determines the new size given the field has been set or updated
+   *
+   * @param index of the field set or updated
+   */
+  protected int calcNewSize(final int index) {
```

Comment:
should probably just call this `calculateNewSize` rather than abbreviate...

Reply:
problem is that in the ssz module seems like "calc" is the normal:
<img width="256" alt="image" src="https://github.com/user-attachments/assets/8b1c943b-4fd7-4e30-b99f-71be3c72f823">

I wanted to maintain the style here. WDYT?

Reply:
ok i guess but i do hate abbreviations...